custom/succeed/data_sources/patient_task_list.json

```diff
@@ -78,7 +78,7 @@
                             "datatype": "string"
                         },
                         "type": "boolean_expression",
-                        "property_value": "true"
+                        "property_value": "True"
```

Comment:
Shouldn't it be `true` (without the quotes)?  JSON supports first-class boolean data types.  Or does this need to be a string for some other reason?